src/main.rs

```diff
@@ -36,7 +36,14 @@ pub mod expansion;
 pub struct Command {
     pub name: &'static str,
     pub help: &'static str,
```

Comment:
You mean make a help function instead of a help string? That could work, but I am not sure when it wouldn't be static.